docs-ref-services/servicebus.md

```diff
@@ -42,76 +42,149 @@ AMQP 1.0 compliant libraries such as Apache Qpid Proton's JMS provider.
 ## Client library
 
 The official Service Bus client is available in [source code form on GitHub](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/servicebus/) and
-binaries and packaged sources [are available on Maven Central](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22azure-servicebus%22).
+binaries and packaged sources [are available on Maven Central](http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22azure-messaging-servicebus%22).
 
-**The [sample code repository](https://github.com/Azure/azure-service-bus/blob/master/samples/Java/) contains samples for:**
-* How to use the [QueueClient](https://github.com/Azure/azure-service-bus/blob/master/samples/Java/azure-servicebus/QueuesGettingStarted/src/main/java/com/microsoft/azure/servicebus/samples/queuesgettingstarted/QueuesGettingStarted.java)
-* How to use the [TopicClient and SubscriptionClient](https://github.com/Azure/azure-service-bus/tree/master/samples/Java/azure-servicebus/TopicsGettingStarted/src/main/java/com/microsoft/azure/servicebus/samples/topicsgettingstarted)
+**The [sample code repository](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus) contains samples for:**
+* How to use the [SendMessage](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/SendMessageAsyncSample.java)
+* How to use the [Processor to receive](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/ServiceBusProcessorSample.java)
 
 [Add a dependency](https://maven.apache.org/guides/getting-started/index.html#How_do_I_use_external_dependencies) to your Maven `pom.xml` file to use the client library in your project. Specify the version as desired.
 
```

Comment:
I believe we should add two sets of links that cover the "where" of the packages. 

If you check out the [guidelines](https://review.docs.microsoft.com/en-us/help/contribute-ref/template-library-index?branch=master) you can see that we have links to the interesting places. samples, github, etc.

Can you add those links? Just for `azure-messaging-servicebus` and `azure-mgmt-servicebus`.

Reply:
I added links for `azure-messaging-servicebus` . And same library can perform management operations.